.github/workflows/forge-stable.yaml

```diff
@@ -45,6 +45,7 @@ jobs:
     outputs:
       IMAGE_TAG: ${{ steps.get-docker-image-tag.outputs.IMAGE_TAG }}
       BRANCH: ${{ steps.determine-test-branch.outputs.BRANCH }}
+      BRANCH_HASH: ${{ steps.hash-branch.outputs.BRANCH_HASH }}
```

Comment:
Why are we using branch hash instead of just branch name?

Is it because the length would exceed the underlying kubernetes namespace length? (we might also wanna enforce that namespace length in the forge.py so we fail earlier)

Reply:
Yeah I was running into issues when the namespace was too long. I can add a comment to document this.

Reply:
or just fail forge.py, otherwise you end up failing later with a strange error

Reply:
Added a comment and an assertion for namespace length to forge.py